packages/frontend/src/ee/pages/AiAgents/AgentPage.tsx

```diff
@@ -205,7 +174,18 @@ const AgentPage = () => {
 
                     {projectUuid && threads && threads.length > 0 && (
                         <Stack gap="xs">
-                            <Title order={6}>Threads</Title>
+                            <Group justify="space-between">
+                                <Title order={6}>Threads</Title>
+                                <Button
+                                    size="compact-xs"
+                                    variant="subtle"
+                                    leftSection={<IconPlus size={16} />}
```

Comment:
what about a write icon or something?

Reply:
Good point. I'm gonna play around with it!

Reply:
I kinda like the dark button better, but it might be too much since we reduce the amount of information here.
The light one works, but I'll need to work on the borders as it's a shade darker than the selects and it kinda annoys me 😅 wdyt?

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/HorFj8gm8QUuuIDE2vgk/87f6b6b2-691d-482d-91e0-55be5ea2d8bb.png)

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/HorFj8gm8QUuuIDE2vgk/09c02381-451c-45fa-95ab-7981c1607e2b.png)